lectures/money_inflation.md

```diff
@@ -33,7 +33,7 @@ The law of motion for the supply of money assumes that the government prints mon
 Our model equates the demand for money to the supply at each time $t \geq 0$.
 
 Equality between those demands and supply gives in a **dynamic** model in which   money supply
-and  price level **sequences** are simultaneously determined by a  set of simultaneous linear  equations.
+and  price level *sequences* are simultaneously determined by a  set of simultaneous linear  equations.
```

Comment:
Thanks @longye-tian .  Can you please also make **dynamic** italic, and also please change "gives in a" to "gives a".